addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiActorPlugHandler.java

```diff
@@ -58,4 +58,18 @@ void parseReport(JsonObject data) {
             updateState(CHANNEL_POWER_CONSUMED, new DecimalType(data.get("power_consumed").getAsBigDecimal()));
         }
     }
+
+    @Override
+    void parseHeartbeat(JsonObject data) {
```

Comment:
looking at the file, i see that this is duplicate code. and power_voltage is actually missing in parseHeartbeat. So maybe parseHeartbeat should just call parseReport. Or add the power_voltage part to parseHeartbeat and call it from parseReport. :)